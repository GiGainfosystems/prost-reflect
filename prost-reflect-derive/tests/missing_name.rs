use prost_reflect_derive::ReflectMessage;

#[derive(ReflectMessage)]
#[prost_reflect(file_descriptor = "FILE_DESCRIPTOR")]
pub struct MyMessage {}

fn main() {}
