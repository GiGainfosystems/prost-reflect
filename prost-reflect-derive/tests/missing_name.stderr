error: at least one of the 'message_name' or 'package_name' arguments is required
 --> tests/missing_name.rs:4:3
  |
4 | #[prost_reflect(file_descriptor = "FILE_DESCRIPTOR")]
  |   ^^^^^^^^^^^^^
